dsmr_exporter/dsmr_exporter.py

```diff
@@ -232,7 +233,7 @@ def run(self):
                         else:
                             input_buffer = s.recv(self.tcp_buffer_size).decode()
                             # todo: rate limit wrong data?
-                            self.logger.debug("DEBUG| read_sockets:{}".format(read_sockets))
+                            self.logger.debug("DEBUG| read_sockets: {}".format(read_sockets))
```

Comment:
Note that this logs the same info as the log before the for loop. So it doesn't really make sense to again log it here each loop iteration.